docs/getting-started/getting-started-with-notebooks.rst

```diff
@@ -0,0 +1,164 @@
+
+Click on File > Save a copy in GitHub to save your notebook to a GitHub repository. You may want to create a new repository for your notebook if you don't already have a repository that you want to save the notebook to.
+
+.. figure:: /assets/images/docs/google-colab-save-on-github.png
+    :width: 35 %
+
+Configure where you want to save the notebook to on GitHub then click OK. Google Colab will push a commit to your repository with the notebook.
+
+.. figure:: /assets/images/docs/google-colab-save-on-github-dialog.png
+    :width: 60 %
+
+.. _no-notebook-in-google-colab:
+
+You have a notebook in GitHub, but it's not in Google Colab
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Click on File > Open notebook then select the GitHub tab in the dialog box. Select the notebook that you want to import from GitHub. This will open the notebook in Google Colab.
+
+.. figure:: /assets/images/docs/google-colab-open-notebook-github.png
+    :width: 60 %
+
+.. _create-dockstore-yml:
+
+Create Your .dockstore.yml File
+-------------------------------
+
+The first step is to create a file named ``.dockstore.yml`` which acts as a configuration file used to describe your notebook. We will cover an example .dockstore.yml written for a simple hello world Jupyter notebook.
+You can also view template .dockstore.yml files :doc:`here </assets/templates/notebooks/notebooks>` that describe the possible fields and values you can use to configure your notebook.
+
+.. include:: /assets/templates/notebooks/example-1-name.yml
+  :code:
+
+Line 1 specifies the ``.dockstore.yml`` version. You should use the latest version, which is version 1.2.
+
+Next is a required key named ``notebooks`` where your notebooks will be configured.
+
+Within this, you need to specify the ``format`` and ``language`` of the notebook. Currently, we only support ``JUPYTER`` notebooks. For a list of all the languages supported, view the :doc:`notebook template files </assets/templates/notebooks/notebooks>`.
+
+Next, you need to specify the absolute path to the notebook file in the Git repository using the ``path`` key.
+
+Finally, you must set ``publish: true`` to publish your notebook in order for you to be able to view your notebook in preview mode. In the future, we will implement a My Notebooks page where you can view your unpublished notebooks.
+
+
+Registering Your Notebook
+-------------------------
+.. include:: /getting-started/github-apps/note--registration.rst
+
+To register your Notebook on Dockstore, you need to install our Dockstore GitHub application on the repository containing your notebook and .dockstore.yml file. By doing so, Dockstore will automatically register the notebooks and update your notebooks whenever your repository is updated on GitHub.
+
+.. include:: /getting-started/github-apps/note--vocabulary.rst
+
+Install the Dockstore GitHub App
+--------------------------------
+.. include:: /getting-started/github-apps/snippet--installation.rst
```

Comment:
Not part of your PR, but I noticed it in reviewing a local build of your branch; in snippet--installation.rst, it's [missing](https://github.com/dockstore/dockstore-documentation/blob/seab-5366/notebooks-tutorial/docs/getting-started/github-apps/snippet--installation.rst?plain=1#L17) an`s` on "Manage Dockstore Installation on GitHub".